src/components/telemetry_monitor/test/application_manager_metric_test.cc

```diff
@@ -85,8 +85,8 @@ TEST(ApplicationManagerMetricWrapper, GetJsonMetric) {
 
 TEST(ApplicationManagerMetricWrapper, GetJsonMetricWithGrabResources) {
   ApplicationManagerMetricWrapper metric_test;
-  utils::ResourseUsage* resources = utils::Resources::getCurrentResourseUsage();
   EXPECT_TRUE(metric_test.grabResources());
+  const utils::ResourseUsage& resources = metric_test.getResources();
```

Comment:
@BSolonenko Don't you have a  :bomb: here?

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/5e15fe123252c49dde19d2d32cd757f1ac7eacf8